kernel/common/get_slice.rs

```diff
@@ -11,27 +11,36 @@ use core::str;
 ///
 /// `foo[..b]` => `foo.get_slice(None, Some(b))`
 ///
-pub trait GetSlice { fn get_slice(&self, a: Option<usize>, b: Option<usize>) -> &Self; }
+pub trait GetSlice {
+    fn get_slice(&self, a: Option<usize>, b: Option<usize>) -> &Self; }
 
 impl GetSlice for str {
     fn get_slice(&self, a: Option<usize>, b: Option<usize>) -> &Self {
         let slice = unsafe { slice::from_raw_parts(self.as_ptr(), self.len()) };
         let a = if let Some(tmp) = a {
             let len = slice.len();
-            if tmp > len { len }
-            else { tmp }
```

Comment:
I actually like keeping these if-else expressions as one-liners here. Is there some reason to need to newline the braces for just one statement?